content/03.results.md

```diff
@@ -166,9 +168,11 @@ We also found evidence of chromothripsis in over 15% of embryonal tumors, ependy
 #### Histologic and oncogenic pathway clustering
 
 UMAP visualization of gene expression variation across brain tumors (**Figure {@fig:Fig4}A**) shows the expected clustering of brain tumors by histology.
-We observed medulloblastomas cluster by molecular subtype with WNT and SHH in distinct clusters and Groups 3 and 4 showing some overlap (**Figure {@fig:FigS4}A**), as expected.
-Additionally, _C11orf95::RELA_ fusion positive ependymomas formed distinct clusters (**Figure {@fig:FigS4}B**).
-However, within low-grade (**Figure {@fig:FigS4}C** and high-grade (**Figure {@fig:FigS4}D** astrocytic tumors, tumors were dispersed by molecular subtype across their respective clusters.
+We observed medulloblastomas cluster by molecular subtype with WNT and SHH in distinct clusters and Groups 3 and 4 showing some overlap (**Figure {@fig:FigS5}A**), as expected.
```

Comment:
Might note that SSH has some serious outliers -  two are not part of any known cluster, and one is in the group 3/4 cluster.

Reply:
Sure, will do. I should look these up specifically because they could then be mis-classified.